action.yml

```diff
@@ -7,7 +7,7 @@ inputs:
     default: 'Code is inconsistent with ITK Coding Style.'
   itk-branch:
     description: 'The Git branch of the ITK repository to fetch .clang-format from.'
-    default: 'release'
+    default: 'v6.0a02'
```

Comment:
This can be overridden in a particular remote module like this:
https://github.com/KitwareMedical/ITKRLEImage/commit/ce7253b9cb712c1e6bb6e07cc0ad56bbf732ea7a